data_analysis/model_ad/notebooks/MG-40_Create_harmonized_pathology_source_file.rmd

```diff
@@ -673,8 +681,17 @@ abca7_final <- join_to_metadata(abca7_individual, abca7_biospecimen, abca7_patho
 
 ## Merge files, write csv file, and upload to synapse
 ```{r}
-pathology_merged <- bind_rows(threex_final, fivex_final, nss_final, abca7_final)
-
+pathology_merged <- bind_rows(
+  threex_final %>% mutate(across(individualID, as.character)),
```

Comment:
Nitpick: I _think_ you just need `mutate(individualID = as.character(individualID)` instead of using `across` for all of these. Though I haven't run this code so... double-check.